vcpkg/consume/third-party-authentication.nuget.md

```diff
@@ -0,0 +1,168 @@
+---
+title: "Authenticate to NuGet package feeds"
+description: Explain authentication methods for accessing NuGet package feeds with vcpkg
+author: vicroms
+ms.author: viromer
+ms.date: 03/03/2025
+ms.topic: tutorial
+---
+# Authenticate to NuGet package feeds
+
+This article explains how to engage vcpkg features that require you to authenticate to NuGet package feeds.
+
+## Binary cache provider: NuGet
+
+NuGet package feeds are supported by many cloud-storage providers, like [Azure Artifacts](<https://www.visualstudio.com/docs/package/nuget/publish>)
+and [GitHub Package registry](<https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-nuget-registry>).
+They also provide access control capabilities, which make them suitable for limiting access within your organization or
+work group.
+
+## Prerequisites
+
+- A NuGet packages feed
+
+## 1 - Add `nuget.exe` to PATH
+
+vcpkg acquires a copy of the `nuget.exe` executable to use for binary caching operations.
+
+Run the `vcpkg fetch nuget` command to obtain the location of the NuGet executable and add it to path.
+
+### Example of `vcpkg fetch nuget` output
+
+```PowerShell
+vcpkg fetch nuget
+C:\dev\vcpkg\downloads\tools\nuget-6.10.0-windows\nuget.exe
+```
+
+Modify your `PATH` environment variable to add the directory containing `nuget.exe`.
+
+### Example setting `PATH` in a PowerShell session
+
+```PowerShell
+$env:PATH="C:\dev\vcpkg\downloads\tools\nuget-6.10.0-windows;$env:PATH"
```

Comment:
```suggestion
$env:PATH="$env:PATH;C:\dev\vcpkg\downloads\tools\nuget-6.10.0-windows"
```

It seems usually better to append when we are just trying to provide a thing rather than override